File: LENTABP10/src/main/java/com/lenta/lentabp10/models/task/WriteOffReason.java
package com.lenta.lentabp10.models.task;

import lombok.Getter;

public class WriteOffReason {

    @Getter private String code;
    @Getter private String name;

    public WriteOffReason(String code, String name){
        this.code = code;
        this.name = name;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/models/task/WriteOffReason.kt
package com.lenta.bp10.models.task


data class WriteOffReason(
        val code: String,
        val name: String,
        val gisControl: String
) {
    companion object {
        fun emptyWithTitle(emptyCategory: String): WriteOffReason {
            return WriteOffReason(code = "", name = emptyCategory, gisControl = "")
        }

        val empty by lazy {
            WriteOffReason(code = "", name = "", gisControl = "")
        }
    }
}